about/people.yml

```diff
@@ -465,12 +465,12 @@ data:
     brown_directory_uuid: 294d8182-b793-46ad-a6a8-6548b7378c2e
     bio: Sanjana is a data science intern working with the Computational Biology Core at CCV. She is concentrating in Computational Biology ('26) and is interested in genetics and public health. She also loves to read sci-fi, run, and spend time with friends!
     image: sanjana_main.jpg
-  - name: Jesus Rodriguez
-    type: Part Time
+  - name: Michelange Senat
```

Comment:
Looks like you've replaced Jesus's entry with yours! You'll need to keep his entry so copy paste it back in.